megameklab/src/megameklab/ui/mek/BMMainUI.java

```diff
@@ -137,8 +137,6 @@ public void createNewUnit(long entityType, boolean isPrimitive, boolean isIndust
         mech.addEngineCrits();
         if (isPrimitive) {
             mech.addPrimitiveCockpit();
-        } else if (isIndustrial) {
```

Comment:
I removed this branch because it is already handled in Mech::addCockpit in the else branch below.